### PR TITLE
Feature/issue 15 inline link to question option change

### PR DESCRIPTION
**Pull request by jaderabbit**
- Inline link from change form 
- Migrations to increase character fields

### CODE REVIEW

File: content/admin.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 from django.contrib import admin
 from django_summernote.admin import SummernoteModelAdmin
 from models import *
```
Comment (smn) [inline content/admin.py]: Not a change in this PR but I'd really recommend you use full module names and explicit imports:

``` python
from content.models import Foo
```

File: content/admin.py [hunk 2]
```diff
@@ -74,3 +75,4 @@ class TestingQuestionOptionAdmin(SummernoteModelAdmin):
 admin.site.register(LearningChapter, LearningChapterAdmin)
 admin.site.register(TestingBank, TestingBankAdmin)
 admin.site.register(TestingQuestion, TestingQuestionAdmin)
+admin.site.register(TestingQuestionOption, TestingQuestionOptionAdmin)
```
Comment (smn) [inline content/admin.py]: nitpick on the newline

File: content/models.py [hunk 3]
```diff
@@ -76,12 +76,16 @@ class Meta:
...
     name = models.CharField(
-        "Name", max_length=50, null=True, blank=False, unique=True)
+        "Name", max_length=500, null=True, blank=False, unique=True)
     question = models.ForeignKey(TestingQuestion, null=True, blank=False)
     order = models.PositiveIntegerField("Order", default=1)
     content = models.TextField("Content", blank=True)
     correct = models.BooleanField("Correct")
 
+    def link(self):
+        return "<a href='/admin/content/testingquestionoption/%s' target='_blank'>Edit</a>" % str(self.id)
```
Comment (smn) [inline content/models.py]: You'd probably want to `reverse()` this?

**Comment by jaderabbit** [comment]
@smn Ready for review.

**Comment by smn** [comment]
:+1:

**Pull request merged**